lib/logstash/outputs/kafka.rb

```diff
@@ -11,18 +11,26 @@
 # 
 # [options="header"]
 # |==========================================================
-# |Kafka Client Version |Logstash Version |Plugin Version |Security Features |Why?
-# |0.8       |2.0.0 - 2.x.x   |<3.0.0 | |Legacy, 0.8 is still popular 
-# |0.9       |2.0.0 - 2.3.x   | 3.x.x |SSL |Works with the old Ruby Event API (`event['product']['price'] = 10`)
-# |0.9       |2.4.x - 5.x.x   | 4.x.x |SSL |Works with the new getter/setter APIs (`event.set('[product][price]', 10)`)
-# |0.10.0.x  |2.4.x - 5.x.x   | 5.x.x |SSL |Not compatible with the <= 0.9 broker
-# |0.10.1.x  |2.4.x - 5.x.x   | 6.x.x |SSL |Not compatible with <= 0.10.0.x broker
+# |Kafka Client Version |Logstash Version |Plugin Version |Why?
+# |0.8       |2.0.0 - 2.x.x   |<3.0.0 |Legacy, 0.8 is still popular 
+# |0.9       |2.0.0 - 2.3.x   | 3.x.x |Works with the old Ruby Event API (`event['product']['price'] = 10`)
+# |0.9       |2.4.x - 5.x.x   | 4.x.x |Works with the new getter/setter APIs (`event.set('[product][price]', 10)`)
+# |0.10.0.x  |2.4.x - 5.x.x   | 5.x.x |Not compatible with the <= 0.9 broker
+# |0.10.1.0  |2.4.x - 5.x.x   | 6.0.0 - 6.1.1 |Not compatible with <= 0.10.0.x broker
+# |0.10.1.1  |2.4.x - 5.x.x   | 6.1.2 - 6.x.x |Not compatible with <= 0.10.1.0 broker
```

Comment:
I'm not sure about this compatibility statement. Added the row and followed the convention in the balte.